core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -481,22 +481,17 @@ private void renderSidebar() {
           boolean horizontalCompact =
               MAX_LENGTH < (3 * sortedWools.size()) + (3 * (sortedWools.size() - 1)) + 1;
           TextComponent.Builder woolText = text();
-          if (!horizontalCompact) {
-            // If there is extra room, add another space to the left of the wools to make them
-            // appear more centered.
-            woolText.append(space());
-          }
-
```

Comment:
> This commit also removes an extra space that was added at the start of the wool line which caused it to start with two spaces rather than the previous one. Looking back at the commits prior to this change and old OCN videos one space seems to be correct.

I believe even in cardinal times, the wools started with 2 spaces in both pgm and cardinal, that's always been a thing